### PR TITLE
sniffnet 1.1.3

### PR DESCRIPTION
**Pull request by p-linnane**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.<details>
  <summary>release notes</summary>
  <pre>### New features

- Added feature to warn you when a newer version of Sniffnet is available on GitHub 🆕 ([#118](https://github.com/GyulyVGC/sniffnet/pull/118))
- Added badge on tab bar to show unread notifications count 🔉
- Added Romanian translation 🇷🇴 ([#113](https://github.com/GyulyVGC/sniffnet/pull/113))

### Improvements

- Introduction of `lazy` widgets to improve the application efficiency ([#122](https://github.com/GyulyVGC/sniffnet/pull/122))
- Aesthetic improvements to create a more modern and minimal UI (use [#119](https://github.com/GyulyVGC/sniffnet/issues/119) to suggest me further improvements)
- Changed keyboard shortcut to open settings from `ctrl+S` to `ctrl+,`, as suggested in [#97](https://github.com/GyulyVGC/sniffnet/issues/97)
- Refactored and cleaned code modules ([#123](https://github.com/GyulyVGC/sniffnet/pull/123))

### Fixes

- Fixed problem that was causing a switch to the initial page when back button was pressed with settings opened on running page and with no packets received
- Fixed problem that was causing application logo to be partially hidden when resizing the window to a lower dimension
- Show `-` option in app protocol picklist only when a filter is active
- Fixed header alignment

### Thanks to:

- @ervinpopescu for Romanian translation ([#113](https://github.com/GyulyVGC/sniffnet/pull/113))
- @0x0177b11f for being my first sponsor ever ❤️ </pre>
</details>

### CODE REVIEW

File: Formula/sniffnet.rb [hunk 1]
```diff
@@ -24,6 +24,7 @@ class Sniffnet < Formula
     depends_on "pkg-config" => :build
     depends_on "alsa-lib"
     depends_on "fontconfig"
+    depends_on "openssl@1.1"
```
Comment (chenrui333) [inline Formula/sniffnet.rb:27]: we need no vendor for such change

**Comment by p-linnane** [comment]
Do we want to scope those additional changes just to Linux? Or move the `openssl@1.1` dep out of the `on_linux do` block?

**Comment by GyulyVGC** [comment]
I'm the maintainer of Sniffnet.
I don't know if this is the best place to ask, but version 1.1.2 installed via Hombrew had some problems (see [this issue](https://github.com/GyulyVGC/sniffnet/issues/109)).
What could be the cause? It seems like the package was built unsuccessfully. I was wondering if the will be solved with this new release.
Thanks in advance!

**Comment by chenrui333** [comment]
@GyulyVGC that looks like openssl issue to me.

**Comment by chenrui333** [comment]
I can reproduce the same issue in my local

```
thread 'main' panicked at '', /Users/rui/Library/Caches/Homebrew/cargo_cache/registry/src/github.com-1ecc6299db9ec823/wgpu-hal-0.14.1/src/metal/adapter.rs:485:13
stack backtrace:
   0:        0x1004888a0 - __mh_execute_header
```

**Comment by chenrui333** [comment]
#128787

**Pull request closed without merging**